tools/wptserve/wptserve/handlers.py

```diff
@@ -237,7 +238,8 @@ def __call__(self, request, response):
         try:
             environ = {"__file__": path}
             sys.path.insert(0, os.path.dirname(path))
-            execfile(path, environ, environ)
+            with open(path) as f:
```

Comment:
Can you make this `open(path, "rb")` so we have the same behaviour on Py2 and 3?